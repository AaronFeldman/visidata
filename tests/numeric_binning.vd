sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/benchmark.csv	o	
	override	numeric_binning	set-option	True		
benchmark	Quantity		type-int		#	
benchmark	Quantity		freq-col		F	
benchmark_Quantity_freq	count		sort-desc		]	
