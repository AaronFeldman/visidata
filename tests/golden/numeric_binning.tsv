Quantity	count	percent	histogram
1.00 - 21.43	44	89.80	**************************************************
21.43 - 41.86	3	6.12	***
41.86 - 62.29	1	2.04	*
123.57 - 144.00	1	2.04	*
62.29 - 82.71	0	0.00	
82.71 - 103.14	0	0.00	
103.14 - 123.57	0	0.00	
